CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## v1.1.0 (2021-08-31)
+
+- Added
+
+  - updates interval changed from `5000ms` to `1000ms`
+  - button for returning to original voice message (only visually, button will be functional in the future)
+
+- Refractoring
+
+  - only needed cache is get from `browser.storage.local`
+  - `elementIsNotAudio` and `elementIsNotImage` changed to `audioElement` and `imageElement` (better readability and consistency)
```

Comment:
IMO this conveys details that are too low-level :smile: I would just call it "refactor variable names"

Reply:
Changes applied! 😄